python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -219,6 +219,85 @@ def convert_convolution(node, **kwargs):
     return [conv_node]
 
 
+@mx_op.register("Deconvolution")
+def convert_deconvolution(node, **kwargs):
+    """Map MXNet's deconvolution operator attributes to onnx's ConvTranspose operator
+    and return the created node.
+    """
+    name, inputs, attrs = get_inputs(node, kwargs)
+
+    num_inputs = len(inputs)
+
+    proc_nodes = kwargs["proc_nodes"]
+    input_node = proc_nodes[kwargs["index_lookup"][inputs[0][0]]].name
+    weights_node = proc_nodes[kwargs["index_lookup"][inputs[1][0]]].name
```

Comment:
input_nodes returned by get_inputs() already has all the node names. Lines 231-236, 247-249 are not required on the latest code. directly pass input_nodes in make_node

Reply:
Done.